enterprise/frontend/src/embedding-sdk/components/private/InteractiveQuestionResult.tsx

```diff
@@ -50,6 +53,10 @@ export const InteractiveQuestionResult = ({
     return <Filter />;
   }
 
+  if (isSummarizeOpen) {
```

Comment:
Could there be a scenario where both summarize and filter is open? If not, I'm wondering if this might be better as a field that indicates what is open (e.g. `'summarize' | 'filter' | ... | null`